### PR TITLE
Fix `InfoBarPanel` namespace, update `InfoBar` to latest WinUI

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Feature

## What is the current behavior?

- `InfoBarPanel` has wrong namespace
- `InfoBar` is not up to date

## What is the new behavior?

- `InfoBarPanel` now has correct namespace
- `InfoBar` is updated to latest WinUI sources
- All `InfoBar` properties and classes have documentation

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBar/InfoBarPanel.Properties.cs [hunk 1]
```diff
@@ -1,40 +1,82 @@
-﻿// MUX reference InfoBarPanel.properties.cpp, commit 533c6b1
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// MUX reference InfoBarPanel.properties.cpp, commit 533c6b1
 
 using Windows.UI.Xaml;
 
-namespace Microsoft.UI.Xaml.Controls
+namespace Microsoft.UI.Xaml.Controls.Primitives
```
Comment (Youssef1313) [inline src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBar/InfoBarPanel.Properties.cs:7]: I might have missed something in the recent symbol matching in the sync generator. It should get caught from there.

File: src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBar/InfoBarTemplateSettings.cs [hunk 2]
```diff
@@ -1,23 +1,38 @@
...
 		{
 			get => (IconElement)GetValue(IconElementProperty);
 			set => SetValue(IconElementProperty, value);
 		}
 
-		public static readonly DependencyProperty IconElementProperty =
+		/// <summary>
+		/// Identifies the InfoBarTemplateSettings.IconElement dependency property.
+		/// </summary>
+		public static DependencyProperty IconElementProperty { get; } =
```
Comment (Youssef1313) [inline src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBar/InfoBarTemplateSettings.cs:35]: This also should have been caught by symbol matching.

Comment (Youssef1313) [reply src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBar/InfoBarTemplateSettings.cs:35]: Oh, actually symbol matching **did** catch this one. It's the PR that's not merged yet: https://github.com/unoplatform/uno/pull/7249/files.

However, the namespace doesn't seem to be caught. So we may want some adjustments.

File: src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBar/InfoBarPanel.Properties.cs (hunk 1 above)
Comment (MartinZikmund) [reply src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBar/InfoBarPanel.Properties.cs:7]: @Youssef1313 Should I create an issue for this one?

Comment (Youssef1313) [reply src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBar/InfoBarPanel.Properties.cs:7]: Maybe. Not a priority though.

**Review by jeromelaban — APPROVED** [review]

**Review by ajpinedam — APPROVED** [review]

**Pull request merged**